### PR TITLE
feature/creating join us component

### PR DESCRIPTION
**Pull request by AlaaSaedy**

### CODE REVIEW

File: frontend/components/JoinUsComponent/index.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <div className="w-3/6 flex flex-col justify-center mb-24">
+          <h1 className=" text-center mt-16 py-5 uppercase text-lg font-black">
+            <Heading type="cardSmall">Join Next Patch</Heading>
+          </h1>
+          <p className="join-us-description text-center px-2 pt-8 text-xs">
+            {description}
+          </p>
+
+          <div className="m-auto mt-8 font-black text-xs uppercase">
+            <Button bgColor="green">Apply As Member</Button>
```
Comment (Tramsi) [inline frontend/components/JoinUsComponent/index.jsx:26]: Don't use static data

File: frontend/components/TeamComponent/index.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Heading } from '../shared/Heading';
+
+const TeamComponent = () => {
```
Comment (Tramsi) [inline frontend/components/TeamComponent/index.jsx:4]: What is this component ?

File: frontend/styles/JoinUs.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.join-us-description {
```
Comment (Tramsi) [inline frontend/styles/JoinUs.css:1]: This css file isn't neccessary, use custom colors from tailwind

File: frontend/components/JoinUsComponent/index.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      <div className="flex justify-around">
+        <div className="my-24 w-1/6">
+          <img
+            className="object-cover w-11/12 hidden md:block"
+            src={left_img}
+            alt=""
+          />
+        </div>
+        <div className="w-3/6 flex flex-col justify-center mb-24">
+          <h1 className=" text-center mt-16 py-5 uppercase text-lg font-black">
```
Comment (Tramsi) [inline frontend/components/JoinUsComponent/index.jsx:18]: Now you can move all styling to Heading component and use the custom sizes and options of heading component

File: frontend/components/JoinUsComponent/index.jsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+import { Heading } from '../shared/Heading';
+import Button from '../shared/Button';
+
+const JoinUs = ({ data }) => {
+  const {
+    title,
+    description,
+    mentorBtn,
+    memberBtn,
+    left_img,
```
[the remark this answers is not in the document]
Comment (AlaaSaedy) [reply frontend/components/JoinUsComponent/index.jsx:11]: previous comments asked to not use any static data inside the component

File: frontend/components/JoinUsComponent/index.jsx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+          <p className="join-us-description text-center px-2 pt-8 text-xs text-c500">
+            {description}
+          </p>
+
+          <div className="mt-8 font-black text-xs text-center uppercase">
+            <Button bgColor="green">{memberBtn}</Button>
+          </div>
+          <div className="mt-5 font-black text-xs text-center uppercase">
+            <Button bgColor="blue">{mentorBtn}</Button>
```
[the remark this answers is not in the document]
Comment (AlaaSaedy) [reply frontend/components/JoinUsComponent/index.jsx:40]: font size, font weight classes aren't working probably as per @Tramsi

File: frontend/components/JoinUsComponent/index.jsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (Nilegfx) [reply frontend/components/JoinUsComponent/index.jsx:11]: I would also encourage getting the images from the backend not as static.

**Pull request closed without merging**